src/update/UpdateManager.cpp

```diff
@@ -130,6 +130,42 @@ void UpdateManager::GetUpdates(bool ignoreNetworkErrors)
 
                 if (code == wxID_YES)
                 {
+                    const audacity::network_manager::Request downloadRequest(mVersionPatch.download.ToStdString());
```

Comment:
This line made me look up what `download` is in VersionPatch.h.  I see it is a wxString.  But that is uninformative.

I like wrapping strings up in classes that indicate the purpose of the string, to prevent mistaken inter-conversions.  And there exists a type URLString, lately added.

Can you use that type in VersionPatch.h?  Can you insert a commit in the sequence before this commit, to make that change?